core/src/main/java/nu/yona/server/rest/HeadersHolder.java

```diff
@@ -0,0 +1,49 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Stichting Yona Foundation This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v.2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *******************************************************************************/
+package nu.yona.server.rest;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.http.HttpHeaders;
+
+public class HeadersHolder
```

Comment:
Instead of just storing all headers, I would prefer to have a more speaking class like `AppInformationHolder`.

Reply:
What about ``PassThroughHeadersHolder``? It holds 'headers', not just 'information', so I'd like to be more specific. At the other hand, it doesn't just hold headers, but the headers that are to be passed through from the incoming requests to the outgoing ones.

Reply:
I still would prefer a domain-centric name with something indicating this is app information instead of a solution-centric name containing something about headers...

Reply:
The class provides the solution of holding headers that are passed through in HTTP requests. Not just app information, but HTTP headers. To me, the class name tells what it does. The earlier name said "it holds headers". The new name is specific about what headers it holds.